style.css

```diff
@@ -1006,3 +1093,1378 @@
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 0;
+    width: 114px;
+    height: 48px;
+    font-family: var(--font4);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 43px;
+    text-align: center;
+    color: #fff;
+    list-style: none;
+    text-decoration: none;
+    justify-content: center;
+  }
+
+  .skill-cont {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 24px;
+    width: 282px;
+    height: 336px;
+    margin-bottom: 130px;
+    margin-top: 130px;
+  }
+
+  .skill-cont .skills {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 8px 12px;
+    gap: 12px;
+    width: 169px;
+    height: 40px;
+    color: white;
+    justify-content: center;
+    font-family: var(--font4);
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .skill-cont ul {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0 12px;
+    gap: 8px;
+    width: 138px;
+    height: 272px;
+  }
+
+  .skill-cont ul li {
+    list-style: none;
+  }
+
+  .skill-cont ul .Codekit a {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 0;
+    width: 114px;
+    height: 48px;
+    font-family: var(--font4);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 43px;
+    text-align: center;
+  }
+
+  .skill-cont ul .Github a {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 0;
+    width: 114px;
+    height: 48px;
+    font-family: var(--font4);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 43px;
+    text-align: center;
+  }
+
+  .skill-cont ul .Codepen a {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 0;
+    width: 114px;
+    height: 48px;
+    font-family: var(--font4);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 43px;
+    text-align: center;
+  }
+
+  .skill-cont ul .Gitlab a {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 0;
+    width: 114px;
+    height: 48px;
+    font-family: var(--font4);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 43px;
+    text-align: center;
+  }
+
+  .skill-cont ul .terminal a {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 0;
+    width: 114px;
+    height: 48px;
+    font-family: var(--font4);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 43px;
+    text-align: center;
+  }
+
+  .form {
+    background: #e5e5e5;
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    padding: 120px 24px 0 24px;
+    width: 1440px;
+    height: 1195px;
+  }
```

Comment:
- [ ] It looks like your form is extending a long height which caused white spaces down the form. What I may suggest is that you don't set the fixed height of your form or just reduce it because it is very high. 

![image](https://user-images.githubusercontent.com/90222110/188200267-b238faaa-d78f-487e-9bea-d4c785a25fcf.png)